commands/bootstrap/resources/topology/topology_single_node.tmpl

```diff
@@ -42,7 +42,7 @@ imports:
 topology_template:
   node_templates:
     JDK:
-      type: org.alien4cloud.java.jdk.linux.nodes.OracleJDK
+      type: org.alien4cloud.java.jdk.linux.nodes.OpenJDK
```

Comment:
Where is this type defined?

Reply:
It's in the zip file tosca_types.zip

Reply:
Yeah but we do relie on <https://github.com/alien4cloud/csar-public-library/blob/45ab6701f8a5193558598f928e8cfc503115989a/org/alien4cloud/java/jdk/linux/types.yml#L15> I think it is a bad idea to rename it.
Furthermore we should port this change to the public csar lib too.

Reply:
I didn't rename it but add a new one. But OK to keep the existing one despite its name.